scss/index.scss

```diff
@@ -1,4 +1,5 @@
 // our project themes
-@import './theme/colors';
+@import "./theme/colors";
```

Comment:
I'd prefer we not make small edits to parts of the code that you are not touching for the feature.

I assume your auto-formatter did this and I will merge it, but we should avoid this in general

Reply:
Gotcha, I will look out for these in the future and revert